ironic-deployment/basic-auth/ironic-rpc-auth-config-tpl

```diff
@@ -0,0 +1,4 @@
+[json-rpc]
+auth_strategy=basic
+username=${IRONIC_USERNAME}
+password=${IRONIC_PASSWORD}
```

Comment:
Note that prior to https://review.opendev.org/742474 the names here were `http_basic_username` and `http_basic_password`. You might want to pass both for a while.